sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -1441,6 +1441,13 @@ class DatasetSuite extends QueryTest with SharedSQLContext {
       assert(e.getCause.isInstanceOf[NullPointerException])
     }
   }
+
+  test("SPARK-23025: Add support for null type in scala reflection") {
+    val data = Seq(("a", null))
+    checkDataset(
+      data.toDS(),
+      data: _*)
```

Comment:
nit: could we make this inlined?

Reply:
sure, since in this file in many places this syntax is used, should I change also other usages accordingly?

Reply:
Nope, let's only fix this one if there are many places to update.